src/couch_httpd_auth.erl

```diff
@@ -346,10 +346,10 @@ handle_session_req(#httpd{method='GET', user_ctx=UserCtx}=Req, _AuthModule) ->
                 ]}},
                 {info, {[
                     {authentication_db, ?l2b(config:get("couch_httpd_auth", "authentication_db"))},
-                    {authentication_handlers, [auth_name(H) || H <- couch_httpd:make_fun_spec_strs(
-                            config:get("httpd", "authentication_handlers"))]}
```

Comment:
It's not possible any more to enable disabled by default auth handlers like [`proxy_authentication_handler`](https://github.com/apache/couchdb-couch/blob/master/src/couch_httpd_auth.erl#L118-L138) or [`null_authentication_handler`](https://github.com/apache/couchdb-couch/blob/master/src/couch_httpd_auth.erl#L115-L116). Also, JavaScript test suite is broken by impossibility to set [`
special_test_authentication_handler`](https://github.com/apache/couchdb-couch/blob/master/src/couch_httpd_auth.erl#L33-L52).